ts/torch_handler/image_classifier.py

```diff
@@ -17,77 +15,24 @@ class ImageClassifier(VisionHandler):
     and returns the name of object in that image.
     """
 
-    TOP_FIVE_CLASSES = 5
-
-    def __init__(self):
-        super(ImageClassifier, self).__init__()
-        self.topk = ImageClassifier.TOP_FIVE_CLASSES
+    topk = 5
+    image_processing = transforms.Compose([
+        transforms.Resize(256),
+        transforms.CenterCrop(224),
+        transforms.ToTensor(),
+        transforms.Normalize(mean=[0.485, 0.456, 0.406],
```

Comment:
Sorry, I didn't quite understand. Do you mean adding a comment?

Reply:
Since this is same for image classifier and segmenter, can you move it as a generic method which returns this to VisionHandler and probably override it in object detector?